### DISCUSSION TITLE
build error for android <HELP>

### DISCUSSION THREAD

**Discussion by Ahsan445** [rustdesk/rustdesk#561] [Q&A] [answered]
Please help i have tried different ways to fix it but noting helped

ahsan@ahsan-VirtualBox:~/Desktop/rusk/rustdesk$ OPENSSL_DIR=/home/ahsan/Desktop/rusk/openssl AARCH64_LINUX_ANDROID_OPENSSL_LIB_DIR=/home/ahsan/Desktop/rusk/openssl VCPKG_ROOT=/home/ahsan/Desktop/rusk/vcpkg ANDROID_NDK_HOME=/home/ahsan/Desktop/rusk/android-ndk-r22b flutter/ndk_arm64.sh
[2022-05-19T15:43:58Z INFO  cargo_ndk::cli] Using NDK at path: /home/ahsan/Desktop/rusk/android-ndk-r22b
[2022-05-19T15:43:58Z INFO  cargo_ndk::cli] NDK API level: 21
[2022-05-19T15:43:58Z INFO  cargo_ndk::cli] Building targets: arm64-v8a
[2022-05-19T15:43:58Z INFO  cargo_ndk::cli] Building arm64-v8a (aarch64-linux-android)
   Compiling magnum-opus v0.4.0 (https://github.com/open-trade/magnum-opus#3c3d0b86)
   Compiling scrap v0.5.0 (/home/ahsan/Desktop/rusk/rustdesk/libs/scrap)
error: failed to run custom build command for `magnum-opus v0.4.0 (https://github.com/open-trade/magnum-opus#3c3d0b86)`

Caused by:
  process didn't exit successfully: `/home/ahsan/Desktop/rusk/rustdesk/target/release/build/magnum-opus-5f167d02c15f1160/build-script-build` (exit status: 101)
  --- stdout
  cargo:info=arm64-android
  cargo:rustc-link-lib=static=opus
  cargo:rustc-link-search=/home/ahsan/Desktop/rusk/vcpkg/installed/arm64-android/lib
  cargo:include=/home/ahsan/Desktop/rusk/vcpkg/installed/arm64-android/include
  rerun-if-changed=/home/ahsan/.cargo/git/checkouts/magnum-opus-7580b098b18e2bc5/3c3d0b8/opus_ffi.h
  rerun-if-changed=/home/ahsan/Desktop/rusk/vcpkg/installed/arm64-android/include

  --- stderr
  /usr/include/stdint.h:26:10: fatal error: 'bits/libc-header-start.h' file not found
  /usr/include/stdint.h:26:10: fatal error: 'bits/libc-header-start.h' file not found, err: true
  thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: ()', /home/ahsan/.cargo/git/checkouts/magnum-opus-7580b098b18e2bc5/3c3d0b8/build.rs:66:18
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
[2022-05-19T15:44:05Z INFO  cargo_ndk::cli] If the build failed due to a missing target, you can run this command:
[2022-05-19T15:44:05Z INFO  cargo_ndk::cli] 
[2022-05-19T15:44:05Z INFO  cargo_ndk::cli]     rustup target install aarch64-linux-android

**Comment by rustdesk** [comment, marked as answer]
https://github.com/rustdesk/rustdesk/issues/565
